### ISSUE TITLE
LAPACKE cmake install config problem

### ISSUE THREAD

**Issue by raovgarimella** [Reference-LAPACK/lapack#185]
If I try to get cmake to find LAPACKE and LAPACK using the installed lapacke-config.cmake file, it cannot find LAPACK from the same location as it found LAPACKE. Looking at the ```lapacke-config-install.cmake.in``` file, I notice it has a suspicious looking line (Line 8)

```
# Load the LAPACK package with which we were built.
set(LAPACK_DIR "${_LAPACKE_PREFIX}/@{LIBRARY_DIR@/cmake/lapack@LAPACK_VERSION@")
```
In particular, the expression ```@{LIBRARY_DIR@``` seems suspect. Is this really how it should be? or should it be ```${LIBRARY_DIR}``` or ```@LIBRARY_DIR@``` or something else?

**Comment by raovgarimella** [comment]
Turns out that when the commit e64606d was made, it did not make the changes to ```LAPACKE/cmake/lapacke-config-install.cmake.in``` and similarly in BLAS. Using ```@CMAKE_INSTALL_LIBDIR@``` as is done in other places in the cmake system fixes the problem.

**Comment by turboencabulator** [comment]
This was fixed in #186 and can be closed, right?

**Comment by raovgarimella** [comment]
Yes
